app/main/views/briefs.py

```diff
@@ -204,6 +206,113 @@ def update_brief_status(brief_id, action):
     return jsonify(briefs=brief.serialize()), 200
 
 
+@main.route('/briefs/<int:brief_id>/award', methods=['POST'])
+def award_brief(brief_id):
```

Comment:
maybe something that mentions pending. `award_pending_brief_response` perhaps. Something less final.